test/linting/lint.sh

```diff
@@ -104,73 +90,40 @@ function _hadolint
 function _shellcheck
 {
   local SCRIPT='SHELLCHECK'
-  local ERR=0
-  local LINT=(shellcheck -x -S style -Cauto -o all -e SC2154 -W 50)
-
-  if ! __in_path "${LINT[0]}"
-  then
-    __log_failure 'linter not in PATH'
-    return 2
-  fi
+  
+  # File paths for shellcheck:
+  F_SH="$(find . -type f -iname '*.sh' \
+    -not -path './test/bats/*' \
+    -not -path './test/test_helper/*' \
+    -not -path './target/docker-configomat/*' -print0 \
+    | xargs -0
+  )"
+  F_BIN="$(find 'target/bin' -perm +111 -type f -or -type l)"
+  F_BATS="$(find 'test' -maxdepth 1 -type f -iname '*.bats')"
```

Comment:
I prefer to be explicit about `test` being a directory. Was there a reason the `/` was dropped?

```suggestion
  F_BATS="$(find 'test/' -maxdepth 1 -type f -iname '*.bats')"
```

Reply:
```
find 'test/' -maxdepth 1 -type f -iname '*.bats'
test//mail_dhparams_default.bats
test//test_helper.bats
test//mail_postfix_inet.bats
test//mail_lmtp_ip.bats
```

Reply:
It is, I was under the impression it might have matched other directories with `test` in the name, but I was mistaken. I don't see any harm in keeping the `/`, but I'm not going to insist on it.